CONTRIBUTING.md

```diff
@@ -19,51 +20,60 @@ Arkouda is an open source project and we love to see new contributors! We welcom
 <a id="issues"></a>
 ## Adding Issues <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
 Most issues fall under the broad categories of bug reports or feature requests.
-If your issue doesn't fit either of these, please add it anyway and provide as much detail as possible
+If your issue doesn't fit either of these, please add it anyway and provide as much detail as possible.
+
+It is always a good idea to review the current issue list and make sure your issue is not already present.
 
 Using github markdown (especially [code blocks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks))
-is very appreciated
+is very appreciated.
 
 <a id="bug-reports"></a>
 ### Bug Reports <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+When reporting a bug, please be sure to include the following information:
 - Summary of Problem
   - What behavior did you observe when encountering the bug?
   - What behavior did you expect to observe?
   - Is this a blocking issue with no known work-arounds?
 - Steps to Reproduce
   - Please provide code that will reproduce the problem if possible.
-Providing simplified programs demonstrating the problem will be appreciated
+Providing simplified programs demonstrating the problem will be appreciated.
 - Configuration Information
-  - What's the output of `ak.get_config()['arkoudaVersion']`
+  - What's the output of `ak.get_config()['arkoudaVersion']`?
 
 <a id="feature-requests"></a>
 ### Feature Requests <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
 Be as specific as possible and provide examples when appropriate. If the requested feature is based on another library 
-(i.e. numpy, pandas, scipy), add a link to their documentation on it
+(i.e. numpy, pandas, scipy), please provide a link to their supporting documentation for reference.
 
 <a id="development"></a>
 ## Developing Arkouda <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
 
 If you don't have anything in mind, check out our [outstanding issues](https://github.com/Bears-R-Us/arkouda/issues) 
 (it's likely a good idea to filter on the label `good first issue`).
-If you find an issue you'd like to work, be sure to comment indicating that! So we avoid someone accidentally duplicating your work
 
-If you already have an idea, be sure to [first add an issue](#issues) detailing the feature or bug fix and
-indicate that you want to work it! This way we can discuss it before you go through the trouble of writing the code
+If you already have an idea for a new feature or have identified a bug, [add an issue](#issues) before you start working on it.
+This way we can discuss it before you go through the trouble of writing the code!
```

Comment:
I would remove this. 

Add a note that we will assign you if you are working the issue.